CMakeLists.txt

```diff
@@ -2,10 +2,12 @@ cmake_minimum_required(VERSION 3.25)
 project(emeralddb VERSION 1.0.0 LANGUAGES C CXX)
 
 # set the C++ standard 
-set(CMAKE_CXX_STANDARD 11)
```

Comment:
cuz we use latest released gtest archive package, some libs in gtest require at least std 14.